src/app/public/modules/text-editor/toolbar/text-editor-toolbar.component.html

```diff
@@ -0,0 +1,329 @@
+
+  <sky-toolbar-item *ngFor="let toolbarSection of toolbarActions"
+    class="sky-text-editor-toolbar-action"
+    [ngClass]="toolbarActionEnum[toolbarSection]"
+  >
+    <ng-container
+      [ngSwitch]="toolbarSection"
+    >
+      <ng-container *ngSwitchCase="toolbarActionEnum.FontFamily">
+        <sky-dropdown
+          class="sky-text-editor-font-picker"
+          label="Font family"
+          [messageStream]="fontPickerStream"
+        >
+          <sky-dropdown-button
+            role="menuitem"
+            [ngStyle]="{
+              'font-family': styleState.font
+            }"
+          >
+            {{ styleStateFontName }}
+          </sky-dropdown-button>
+          <sky-dropdown-menu>
+            <sky-dropdown-item *ngFor="let fontModel of fontList">
+              <button
+                class="menu-dropdown-item"
+                type="button"
+                [ngStyle]="{
+                  'font-family': fontModel.value
+                }"
+                (click)="execCommand('fontname', fontModel.name)"
+              >
+                {{ fontModel.name }}
+              </button>
+            </sky-dropdown-item>
+          </sky-dropdown-menu>
+        </sky-dropdown>
+      </ng-container>
+      <ng-container *ngSwitchCase="toolbarActionEnum.FontSize">
+        <sky-dropdown
+          class="font-size"
+          label="Font size"
+          [messageStream]="fontSizeStream"
+        >
+          <sky-dropdown-button
+            role="menuitem"
+            [ngStyle]="{
+              'font-family': styleState.fontSize
+            }"
+          >
+            {{ styleState.fontSize + 'px'  }}
+          </sky-dropdown-button>
+          <sky-dropdown-menu>
+            <sky-dropdown-item *ngFor="let size of fontSizeList">
+              <button
+                class="menu-dropdown-item"
+                type="button"
+                (click)="changeFontSize(size)"
+              >
+                {{ size + 'px' }}
+              </button>
+            </sky-dropdown-item>
+          </sky-dropdown-menu>
+        </sky-dropdown>
+      </ng-container>
+      <ng-container *ngSwitchCase="toolbarActionEnum.FontStyle">
+        <div
+          class="sky-switch-icon-group"
+        >
+          <sky-checkbox
+            icon="bold"
+            label="Bold"
+            title="Bold"
+            [ngModel]="styleState.boldState"
```

Comment:
Should we be using reactive forms? That is our recommendations to our consumers but we are using a template form here.

Reply:
Agreed. Decided to defer this for the beta release. Will create a bug to fix.